README.md

```diff
@@ -362,6 +362,45 @@ The `SKIPPED` result code indicates that a given test was explicitly skipped by
 * taking a very long time to run
 * failing or passing for the incorrect reason
 
+### Debugging individual tests
+
+While `stl-lit` is super awesome in finding out that *something* is wrong or not even compiling, it is not really
+helpful in debugging *what* is going wrong. However, debugging individual tests is rather simple given some additional
+steps. Let's assume we want to debug a new feature with tests located in `tests\std\tests\GH_XXXX_meow`
+
+As always, build the STL from your branch and run the tests:
+```
+C:\STL\out\build\x64> ninja
+C:\STL\out\build\x64> python tests\utils\stl-lit\stl-lit.py -v C:\STL\tests\std\tests\GH_XXXX_meow
+```
+
+Let's assume one of the tests fails an assert and we want to debug that configuration. `stl-lit` will conveniently print
+the build command, which is far too long to provide here in full. The important part is to add the following option to
+provide debug symbols: `/Zi /Fdbark.pdb`.
+
+You can replace `bark` with any descriptive name you like. Add these before the `"-link"` option in the command line
+and recompile.
+Example:

```

Comment:
This doesn't start a new paragraph, so it should be fused into the previous line (so the file looks like the rendered Markdown).